gettsim/taxes/zve.py

```diff
@@ -126,11 +130,13 @@ def kinderfreibetrag(tax_unit, tb):
     # For non married couples or couples where both earn enough this are a lot easier.
     # Just split the kinderfreibetrag 50/50.
     else:
-        tax_unit.loc[~tax_unit["child"], "kifreib"] = 0.5 * tb["kifreib"] * child_num_kg
+        tax_unit.loc[~tax_unit["child"], "kifreib"] = (
+            0.5 * params["kifreib"] * child_num_kg
+        )
         return tax_unit
 
 
-def zve_nokfb(tax_unit, tb):
+def zve_nokfb(tax_unit, params):
```

Comment:
Do we use `params`? If not, delete unless there are hardcoded numbers in there that we will remove soon (I only see the diff without the body).

Reply:
It is in there because we assign the function in the dictionary and therefor I think every possible function needs the same interface. `zve_abg_nokfb`uses params.

Reply:
I do not understand this comment.

Reply:
Which dictionary, where?

Reply:
Got it.